package.json

```diff
@@ -78,7 +78,7 @@
     "vue-plugin-helper-decorator": "^0.0.12",
     "vue-property-decorator": "^8.5.1",
     "vue-template-compiler": "2.6.12",
-    "vue-test-utils-helpers": "https://github.com/bcgov/vue-test-utils-helpers.git",
+    "vue-test-utils-helpers": "github:bcgov/vue-test-utils-helpers",
```

Comment:
We will have to see if the CI/CD actions are OK with this. I assume you've run the tests locally, ie, this package was retrieved correctly using this spec?

Reply:
i applied a git fetch upstream, git rebase and push back to origin.

should be ready to build

Reply:
Did you run `npm audit fix` here also?

Reply:
Oh, I see a commit that says you did. ✔️